portal-web/test/functional/com/liferay/portalweb/paths/portlet/remoteapps/RemoteAppsEntry.path

```diff
@@ -20,6 +20,16 @@
 	<td>//input[contains(@id,'customElementURLs')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>HTML_ELEMENT_INPUT</td>
+	<td>//input[contains(@id,'customElementHTMLElementName')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CSS_ELEMENT_INPUT</td>
+	<td>//input[contains(@id,'customElementCSSURLs')]</td>
+	<td></td>
+</tr>
```

Comment:
This is usually good xpath, but my preference is to use some common xpaths found in liferay-portal/portal-web/test/functional/com/liferay/portalweb/paths/pathlib such as TextInput#ANY. I'd prefer we don't create more xpaths if we have a common pattern that can fit in well.

Please see as example: https://github.com/liferay/liferay-portal/blob/7eac9cd5daa98add4511fc853a91cd41e8f6eec6/portal-web/test/functional/com/liferay/portalweb/macros/RemoteAppsEntry.macro#L4-L18